.storybook/config.js

```diff
@@ -11,10 +11,10 @@ import { addReadme } from 'storybook-readme';
 addDecorator(withInfo({
   source:false,
 }));
-addDecorator(addReadme);
+// addDecorator(addReadme);
 addDecorator(withA11y);
 addDecorator(withKnobs);
-addDecorator(withViewport);
+// addDecorator(withViewport);
```

Comment:
I was having an issue with the story container being pushed to the left out of view. Removing this fixed it. Need to come up with another solution

Reply:
Have you tried manually setting the size of your story's container element? I've had to do this.